static/app/views/releases/detail/commits.tsx

```diff
@@ -51,10 +51,12 @@ class Commits extends AsyncView<Props, State> {
     };
   }
 
-  componentDidUpdate(prevProps: Props) {
+  componentDidUpdate(prevProps: Props, prevContext) {
```

Comment:
is there a type we can use for the prevContext? maybe just: `Record<string, any>` ?